android/src/main/java/com/onegini/mobile/sdk/reactnative/exception/OneginiWrapperErrors.kt

```diff
@@ -9,4 +9,5 @@ enum class OneginiWrapperErrors(val code: String, val message: String) {
     AUTHENTICATOR_DOES_NOT_EXIST("8006", "The Fingerprint authenticator does not exist"),
     PROFILE_DOES_NOT_EXIST("8007", "The profileId does not exist"),
     USER_PROFILE_IS_NULL("8008", "The user profile is null"),
+    PARAMETERS_NOT_CORRECT("8009", "The parameters to the called function are not correct")
```

Comment:
As mentioned in previous comment, native SDK will check if pin is null, so this error shouldn't be needed